### DISCUSSION TITLE
Alignment issues when rows are merged

### DISCUSSION THREAD

**Discussion by giorgiopaulon** [davidgohel/flextable#518] [Q&A]
I am using `merge_v()` to avoid repeated values in a flextable column. However, when I do so, the alignment of the column is drastically changed and it seems that autofit() does not have the expected behaviour. Below are examples with and without the call to `merge_v()`.

**Note**: this behavior occurs only when exporting / embedding into a .pdf. When I visualize the table within Rstudio, everything appears to be correct. Also, if anyone knows how to render math symbols (`\leq` instead of <=) when the desired output is .pdf, please let me know. 

```{r}
tibble(Name = rep(c('Example', 'Example (<= 8h)', 'Example (> 8h)'), each = 2), 
       Value = c(0, 2, 1, 3, 2, 4), 
       `Value 2` = c(1, 1, 2, 2, 3, 3)) %>% 
  flextable() %>% 
  bold(part = "header") %>% 
  fix_border_issues(part = "all") %>% 
  align(align = "center", part = 'all') %>% 
  align(j = 1, align = "left", part = 'all')
```

![Screenshot 2023-03-10 at 1 25 30 PM](https://user-images.githubusercontent.com/90702867/224408184-b7fdafd8-ff56-4315-8dc1-37bc70a8f277.png)

```{r}
tibble(Name = rep(c('Example', 'Example (<= 8h)', 'Example (> 8h)'), each = 2), 
       Value = c(0, 2, 1, 3, 2, 4), 
       `Value 2` = c(1, 1, 2, 2, 3, 3)) %>% 
  flextable() %>% 
  merge_v(j = c(1, 3)) %>%
  bold(part = "header") %>% 
  fix_border_issues(part = "all") %>% 
  align(align = "center", part = 'all') %>% 
  align(j = 1, align = "left", part = 'all')
```
![Screenshot 2023-03-10 at 1 24 14 PM](https://user-images.githubusercontent.com/90702867/224408298-b461ec86-c4f2-4e30-b7b7-d9fba3a8ce0a.png)

**Comment by davidgohel** [comment]
For Q1, add a call to `autofit()`

For Q2, sorry, flextable does not manage math symbols. We provide `as_equation()` but in your case, you want an automatic substitution and that's not implemented

**Reply by giorgiopaulon** [reply]
Using `autofit()` as suggested does not completely resolve the issue.

```
tibble(Name = rep(c('Example', 'Example (<= 8h)', 'Example (> 8h)'), each = 2), 
       Value = c(0, 2, 1, 3, 2, 4), 
       `Value 2` = c(1, 1, 2, 2, 3, 3)) %>% 
  flextable() %>% 
  merge_v(j = c(1, 3)) %>%
  bold(part = "header") %>% 
  fix_border_issues(part = "all") %>% 
  align(align = "center", part = 'all') %>% 
  align(j = 1, align = "left", part = 'all') %>% 
  autofit()
```

produces 

![Screenshot 2023-03-14 at 1 21 49 PM](https://user-images.githubusercontent.com/90702867/225101534-d8f04d21-bfa3-4a4c-bb92-916c3abc6188.png)

The alignment of the merged cells is not correct, and it also seems like a lot of padding (white space) is added between columns.

**Reply by davidgohel** [reply]
ok, I can see it now! Yes, you're right, there is a bug with merged cells
